api/client/options.proto

```diff
@@ -29,6 +39,8 @@ message CommandLineOptions {
   uint64 burst_size = 10;
   // See :option:`--address-family` for details.
   string address_family = 11;
+  // See :option:`--request_options` for details.
+  RequestOptions request_options = 12;
```

Comment:
Is the idea that these will be delivered dynamically in the future in the same format as well? How do you think this will work with non-uniform, will this be `oneof` with some non-uniform proto config?

Reply:
Changed this to have a `oneof` right off the bat.